### ISSUE TITLE
Request save data decryption fature

### ISSUE THREAD

**Issue by Zero-G-Sys** [Sinflower/UberWolf#1]
Requesting save data decryption feature. It would be nice to have a way to decrypt, modify and encrypt a savefile.

I particularly want a way to export strings out of the game, but the engine only lets you save a variable or string into a savefile. So I tried to decrypt that to get my exported string to no avail.

**Comment by Sinflower** [comment]
Thank you for the suggestion, I will look into it and get back to you as soon as I have results.

**Comment by Sinflower** [comment]
Hi,
It took longer than expected, but I finally managed to parse the wolf save format fully.
Attached is the save decrypted I wrote. Just drop a `.sav` file onto the exe, and it will create a `.json` file containing all information stored in the save game. Just to let you know, a lot of data is stored in the save file, meaning it can be hard to find the values you are looking for. To convert the JSON back to a save file, just drop it onto the exe, and it will create a `.sav_new`, which, once renamed to .sav, should be readable by the respective game.
I did some simple testing, meaning if it does not work, feel free to post the save file together with the game name, and I will take a look.

[WolfSave.zip](https://github.com/Sinflower/UberWolf/files/14316741/WolfSave.zip)

Edit: The tools repo is now online and can be found here: https://github.com/Sinflower/WolfSave
